code/controllers/subsystem/parallax.dm

```diff
@@ -15,8 +15,8 @@ SUBSYSTEM_DEF(parallax)
 	/// Weighted list with the parallax layers we could spawn
 	var/random_parallax_weights = list(
 		/atom/movable/screen/parallax_layer/random/space_gas = 35,
-		/atom/movable/screen/parallax_layer/random/asteroids = 35,
-		PARALLAX_NONE = 30,
+		/atom/movable/screen/parallax_layer/random/asteroids = 0,
+		PARALLAX_NONE = 0,
```

Comment:
What's up with this change? Is this a debug value that got committed?

Reply:
Fuck, yeah it is, I was testing the camo